homeassistant/components/stream/core.py

```diff
@@ -97,6 +106,7 @@ def __init__(
         self._idle_timer = idle_timer
         self._cursor = None
         self._event = asyncio.Event()
+        self._part_event = asyncio.Event()
```

Comment:
The previous event is used to signal when Segments are ready. We need a new event to signal when the Parts are pushed.